cmd/syncer/helm/converter.go

```diff
@@ -55,7 +56,7 @@ func main() {
 	ctx = klog.NewContext(ctx, logger)
 	_ = ctx
 	fs.VisitAll(func(flg *pflag.Flag) {
-		logger.V(1).Info("Command line flag", flg.Name, flg.Value)
+		logger.V(1).Info(fmt.Sprintf("Command line flag %s %s", flg.Name, flg.Value))
```

Comment:
Sadly, the linter is overly picky about this.  You can tell because the linter output in this case explicitly tells you that there is a list of accepted complaints that you can update; see https://github.com/kcp-dev/edge-mc/blob/4219d05658f25bc36f0d01d4b57f8266bd566f2c/hack/verify-contextual-logging.sh#L63.  See the exceptions already listed in https://github.com/kcp-dev/edge-mc/blob/4219d05658f25bc36f0d01d4b57f8266bd566f2c/hack/logcheck.out .